fe/fe-common/src/main/java/org/apache/doris/catalog/Type.java

```diff
@@ -149,6 +149,10 @@ public abstract class Type {
         numericDateTimeTypes = Lists.newArrayList();
         numericDateTimeTypes.add(DATE);
         numericDateTimeTypes.add(DATETIME);
```

Comment:
please add regession test for creating inverted index using  datev2，datetimev2，decimalv3，

Reply:
@xiaokang add regression cases in this PR:https://github.com/apache/doris/pull/19843